dotcom-rendering/src/web/components/Standfirst.tsx

```diff
@@ -146,6 +146,11 @@ const standfirstStyles = (format: ArticleFormat, palette: Palette) => {
 						margin-bottom: ${space[3]}px;
 						max-width: 540px;
 						color: ${palette.text.standfirst};
+
+						a:hover {
+							border-bottom: solid 1px
+								${palette.hover.standfirstLink};
+						}
```

Comment:
We could maybe move this outside this switch statement for `standfirstStyles` and have it in its own, hoverStyles style?

That way, in decidePalette, we can put the complete logic and it will perhaps make more sense? So, instead of having a check on design here and selectively using hover colour. You can put the check on Design inside `hoverStandfirstlink` and only return when Design === blog.

That make sense? Shout if you want to pair on this!